spark-lint/src/main/kotlin/com/adevinta/spark/lint/LintIssueRegistry.kt

```diff
@@ -36,5 +36,7 @@ public class LintIssueRegistry : IssueRegistry() {
     override val api: Int = CURRENT_API
     override val issues: List<Issue> = listOf(
         MaterialComposableUsageDetector.ISSUE,
+        UnknownStringResourceAnnotationsDetector.UNKNOWN_ANNOTATION_NAME_ISSUE,
+        UnknownStringResourceAnnotationsDetector.UNKNOWN_ANNOTATION_VALUE_ISSUE,
```

Comment:
I wasn't sure whether to go for
- 1 detector and 2 issues
- 2 detectors and 1 issue each
- 1 detector and 1 issue

I felt like having 2 issues for a single detector is fine, let me know what you think!